cost/google/old_snapshots/google_delete_old_snapshots.pt

```diff
@@ -186,9 +188,12 @@ policy "policy_old_snapshots" do
     escalate $esc_approve_delete_snapshots
     export do
       resource_level true
-      field "projectId" do
+      field "projectID" do
         label "Project ID"
       end
+      field "projectName" do
+        label "Project Name"
+      end
```

Comment:
Should we consider scraping recommendations for this "Google Old Snapshots" policy?

Reply:
Not scraping from this policy until it can provide accurate cost data.